test/src/server/routes/entity/entity.js

```diff
@@ -80,3 +84,436 @@ describe('getDefaultAliasIndex', () => {
+		// Create Entities
+		let work = await createWork();
+
+		// Create Relation and its data between two entites
+		const relationshipData = {
+			id: 1,
+			sourceBbid: author.toJSON().bbid,
+			targetBbid: work.toJSON().bbid,
+			typeId: 1
+		};
+		const relationshipTypeData = {
+			description: 'test descryption',
+			id: 1,
+			label: 'test label',
+			linkPhrase: 'test phrase',
+			reverseLinkPhrase: 'test reverse link phrase',
+			sourceEntityType: 'Author',
+			targetEntityType: 'Work'
+		};
+		relationshipTypeData.id = random.number();
+		await new RelationshipType(relationshipTypeData)
+			.save(null, {method: 'insert'});
+		relationshipData.typeId = relationshipTypeData.id;
+		relationshipData.id = random.number();
+		const relationship = await new Relationship(relationshipData)
+			.save(null, {method: 'insert'});
+
+		// Create Relationship Set for Author Entity
+		const authorRelationshipSetId = random.number();
+		let authorRelationshipSet = await new RelationshipSet({id: authorRelationshipSetId})
+			.save(null, {method: 'insert'})
+			.then(
+				(model) =>
+					model.relationships().attach([relationship]).then(() => model)
+			);
+		authorRelationshipSet = await authorRelationshipSet.refresh({withRelated: 'relationships'});
+		const authorRelationshipSetJSON = authorRelationshipSet.toJSON();
+
+		// Create Relationship Set for Work Entity
+		const workRelationshipSetId = random.number();
+		let workRelationshipSet = await new RelationshipSet({id: workRelationshipSetId})
+			.save(null, {method: 'insert'})
+			.then(
+				(model) =>
+					model.relationships().attach([relationship]).then(() => model)
+			);
+		workRelationshipSet = await workRelationshipSet.refresh({withRelated: 'relationships'});
+		const workRelationshipSetJSON = workRelationshipSet.toJSON();
+
+		// Create a Revision for Entities
+		const revisionId = random.number();
+		await new Revision({authorId: editorJSON.id, id: revisionId})
+			.save(null, {method: 'insert'});
+
+		// Set the Entity Attributes as Required
+		author.set('revisionId', revisionId);
+		author.set('relationshipSetId', authorRelationshipSetJSON.id);
+		work.set('revisionId', revisionId);
+		work.set('relationshipSetId', workRelationshipSetJSON.id);
+
+		// Save the Entities and Update the database
+		await author.save(null, {method: 'update'});
+		await work.save(null, {method: 'update'});
+
+		// Fetch Refreshed Entities from database
+		author = await new Author({bbid: author.get('bbid')}).fetch({withRelated: 'relationshipSet.relationships'});
+		work = await new Work({bbid: work.get('bbid')}).fetch({withRelated: 'relationshipSet.relationships'});
+
+		// Get Updated Work
+		// fetchedWork = await new Work({bbid: work.get('bbid')}).fetch();
+		const affectedEntities = await deleteRelationships(orm, null, author.toJSON());
+
+		expect(affectedEntities).to.have.length(1);
```

Comment:
You're missing an expectation, looking at the test title:
`expect(workRelationshipSetJSON.id).to.not.equal(affectedEntities[0].get('relationshipSetId')`